### DISCUSSION TITLE
Kill logs

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#3129] [Ideas]
I would like the kill log to be printed to console or saved in a txt file. 
It would also be nice if we could run code on a player or in general when a player attacks another player. It will also be important to know if the attacker has killed the victim player in godmode or not. Something like this

John_Doe69 (248403280) killed Jane_Doe (180808340)
and something like this for a godmode kill
GODMODE) John_Doe69 (248403280) killed Jane_Doe (180808340)

I'm not asking for something complex, just some basic kill logs.

**Comment by lonelybud** [comment]
i have this build in my fork but it requires breaking changes. Or you could find some easy way to do this.
